Import TUR Action's GPG key if we need to download dependencies.
--- a/build-package.sh
+++ b/build-package.sh
@@ -459,6 +459,10 @@
 		gpg --import "$TERMUX_SCRIPTDIR/packages/termux-keyring/termux-autobuilds.gpg"
 		gpg --no-tty --command-file <(echo -e "trust\n5\ny")  --edit-key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
 	}
+	gpg --list-keys 43EEC3A2934343315717FF6F6A5C550C260667D1 > /dev/null 2>&1 || {
+		gpg --import "$TERMUX_SCRIPTDIR/packages/tur-repo/tur.gpg"
+		gpg --no-tty --command-file <(echo -e "trust\n5\ny")  --edit-key 43EEC3A2934343315717FF6F6A5C550C260667D1
+	}
 fi
 
 for ((i=0; i<${#PACKAGE_LIST[@]}; i++)); do
